observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -46,7 +46,7 @@ The following advisory is available for the Network Observability Operator 1.6.0
 
 [IMPORTANT]
 ====
-Before upgrading to the latest version of the Network Observability Operator, you must xref:../../observability/network_observability/installing-operators.adoc#network-observability-updating-migrating_network_observability[Migrate removed stored versions of the FlowCollector CRD]. An automated solution to this workaround is planned with https://issues.redhat.com/browse/NETOBSERV-1747(*NETOBSERV-1747*).
+Before upgrading to the latest version of the Network Observability Operator, you must xref:../../observability/network_observability/installing-operators.adoc#network-observability-updating-migrating_network_observability[Migrate removed stored versions of the FlowCollector CRD]. An automated solution to this workaround is planned with (link:https://issues.redhat.com/browse/NETOBSERV-1747[*NETOBSERV-1747*]).
```

Comment:
Hi @skrthomas in the preview, there is an extra right parenthesis before the Jira label, but I can't figure where it comes from..there are 2 parens in the content:
![image](https://github.com/user-attachments/assets/2b1adcc4-8c85-4117-b14f-a23ea2d0f48b)